mofa/simulation/raspa2.py

```diff
@@ -0,0 +1,291 @@
+from dataclasses import dataclass
+import subprocess
+import os
+from pathlib import Path
+import shutil
+import numpy as np
+import ase
+
+_file_dir = Path(__file__).parent / "files" / "raspa2_template"
+
+
+@dataclass
+class RASPA2Runner:
+    """Interface for running pre-defined RASPA2 workflows."""
+
+    raspa2_command: str = ""
+    """Invocation used to run RASPA2 on this system"""
+
+    run_dir: Path = Path("raspa2-runs")
+    """Path to store RASPA2 files"""
+
+    def _calculate_cell_size(
```

Comment:
These seem the same functions from the gRASPA runner? Not critical now, but we should make sure to have all the runners call the same functions in #157